app/src/components/main/tooltips/textTooltip/textTooltip.jsx

```diff
@@ -1,15 +1,23 @@
+import React from 'react';
 import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
 import styles from './textTooltip.scss';
 
 const cx = classNames.bind(styles);
 
-export const TextTooltip = ({ tooltipContent }) => (
-  <div className={cx('text-tooltip')}>{tooltipContent}</div>
+const getContent = (content) =>
+  content.split(/(?:\r\n|\r|\n)/g).map((str, key) => <p key={String(key)}>{str || <br />}</p>);
+
+export const TextTooltip = ({ tooltipContent, withLineBreaks }) => (
+  <div className={cx('text-tooltip')}>
+    {withLineBreaks ? getContent(tooltipContent) : tooltipContent}
```

Comment:
It can be solved with css only:
Add `white-space: pre-wrap;` to class `.tooltip-content` in:
app/src/components/main/tooltips/hoverableTooltip/hoverableTooltip.scss#48
app/src/components/main/tooltips/tooltip/tooltip.scss#61

Reply:
Okay. I'll refactor code using pure css soon. Usually customer's wish list begins with addind line breaks support and continues with maintaining leading spaces, followed by wrapping content with `react-dotdotdot`; so, I usually don't use css only solution and prefer to start with html-markup injection.

Reply:
Fixed.